src/components/manage/Contents/Contents.jsx

```diff
@@ -1275,7 +1332,7 @@ class Contents extends Component {
                       <Table.HeaderCell
                         width={Math.ceil(16 / this.state.index.selectedCount)}
                       >
-                        <FormattedMessage id="Title" defaultMessage="Title" />
+                        <FormattedMessage id="Title" defaultMessage="NAME" />
```

Comment:
Why?

Reply:
Because It is mention in Pastanaga style.